terraform/modules/quorum-cluster-region/variables.tf

```diff
@@ -387,3 +401,67 @@ variable "observer_node_counts" {
     sa-east-1      = 0
   }
 }
+
+variable "backup_lambda_binary" {
+  description = "Name of BackupLambda binary"
+  default = "BackupLambda"
+}
+
+variable "backup_lambda_binary_url" {
+  description = <<DESCRIPTION
+URL to retrieve the backup lambda binary.
+DESCRIPTION
+  default     = "https://github.com/EximChua/BackupLambda/releases/download/0.1/BackupLambda.zip"
+}
+
+# this is the lambda zip, must be a relative path
+# eg "BackupLambda.zip"
+variable "backup_lambda_output_path" {
+  description = "Relative path to the BackupLambda zip"
+  default = "BackupLambda.zip"
+}
+
+# This is the full path to the private key 
+# eg "/Users/xxxx/.ssh/cert"
+variable "private_key_path" {
+  description = "Full path to the private key to access Quorum nodes"
+  default = ""
+}
+
+# This is the public key path
+# eg "/Users/xxxx/.ssh/cert.pub"
+variable "public_key_path" {
+  description = "Full path to the public key for Quorum nodes"
+  default = ""
+}
+
+# output prefix of encrypted SSH key, region will be appended to the filename
+variable "enc_ssh_path" {
+  description = "Full path to the encrypted SSH key to be generated, region will be appended to the filename"
+  default = "enc-ssh"
+}
+
+# key on S3 bucket
+variable "enc_ssh_key" {
+  description = "The key to access the encrypted SSH key on the S3 bucket"
+  default = "enc-ssh"
+}
+
+# If singular, it's 1 hour, 1 minute, 1 week, 1 day, etc.
+variable "backup_interval" {
+  description = <<DESCRIPTION
+Schedule expression for backup event, see https://docs.aws.amazon.com/lambda/latest/dg/tutorial-scheduled-events-schedule-expressions.html for examples.
+cron(45 08 ? * WED *) will trigger every WED at 8 45 am GMT
+cron(05 13 ? * MON *) will trigger every MON at 1 05 pm GMT
+rate(3 mins) will trigger every 3 minutes
+rate(7 hours) will trigger every 7 hours
+DESCRIPTION
+  default     = "rate(4 hours)"
+}
+
+variable "backup_enabled" {
+  description = <<DESCRIPTION
+Enable backup of chain data.
+DESCRIPTION
+  default = "true"
```

Comment:
nit: default to `false` to make our lives easier when testing

Reply:
Fixed